0.10/README.md

```diff
@@ -85,12 +85,23 @@ Repository organization
 Environment variables
 ---------------------
 
-To set environment variables, you can place them as a key value pair into a `.sti/environment`
-file inside your source code repository.
+Application developers can use the following environment variables to configure the runtime behavior of this image:
+
+NAME        | Description
+------------|-------------
+NODE_ENV    | NodeJS runtime mode (default: production)
+DEV_MODE    | When set to "true", use `nodemon` to automatically reload the server while you work (default: false)
+NPM_RUN     | Select an alternate / custom runtime mode, defined in your `package.json` file's [`scripts`](https://docs.npmjs.com/misc/scripts) section (default: npm run "start")
+HTTP_PROXY  | set the npm proxy to use during assembly
+HTTPS_PROXY | set the npm proxy to use during assembly
+
+One way to define a set of environment variables is to include them as key value pairs in your repo's `.sti/environment` file.
```

Comment:
it's .s2i/environment now (both work but .sti is deprecated so we might as well fix the doc)